lms/models/lti_user.py

```diff
@@ -28,6 +28,12 @@ class LTIUser:  # pylint: disable=too-many-instance-attributes
     application_instance_id: int
     """ID of the application instance this user belongs to"""
 
+    context_id: str
+    """ID of the course in the LMS, context_id using LTI naming"""
+
+    resource_link_id: Optional[str] = None
```

Comment:
These are the raw values not access to the SQLA models here, that's trickier to achieve without a big refactor.